themes/classic/_dev/js/product.js

```diff
@@ -104,7 +104,7 @@ $(document).ready(function () {
       max: 1000000
     });
 
-    quantityInput.on('change', function (event) {
+    quantityInput.on('keyup change', function (event) {
```

Comment:
Ideally I think that the "updatedProductQuantity" event shouldn't be triggered unless the product quantity has effectively changed.
You can keep `keyup change` but you'll have to check if the value of quantityInput has actually changed, so that you don't trigger it twice:

```js
var quantity = quantityInput.val();
quantityInput.on('keyup change', function (event) {
  const newValue = $(this).val();
  if (newValue !== quantity) {
    quantity = newValue;
    // trigger events only when value changes
    // ...
  }
});
```